tests/dummy/app/routes/index.js

```diff
@@ -4,7 +4,7 @@ import FaIconsMixin from 'dummy/mixins/faicons';
 
 export default Ember.Route.extend({
   model: function() {
-    return Ember.Object.createWithMixins(FaIconsMixin,{
+    return Ember.Object.extend(FaIconsMixin).create({
```

Comment:
Mixin can likely be removed here and can be part of the view backing this route for now